projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-settings-page.php

```diff
@@ -109,7 +165,7 @@ function page_render() {
 					</div>
 					<div class="manage-left" style="width: 100%;">
 						<form class="jetpack-modules-list-table-form" onsubmit="return false;">
-						<table class="<?php echo implode( ' ', $list_table->get_table_classes() ); ?>">
+						<table class="<?php echo implode( ' ', $list_table->get_table_classes() ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Escaped in class.jetpack-modules-list-table.php ?>">
```

Comment:
There's no escaping there. There isn't really any need for it either, but I'd go ahead and do `esc_attr()` here anyway.
```suggestion
						<table class="<?php echo esc_attr( implode( ' ', $list_table->get_table_classes() ) ); ?>">
```